### PR TITLE
Setup eslint and fix current issues

### PR DESCRIPTION
**Pull request by ozasadnyy**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -10,46 +12,51 @@ const mergeStream = require('merge-stream');
...
 const history = require('connect-history-api-fallback');
+const eslint = require('gulp-eslint');
+const friendlyFormatter = require('eslint-friendly-formatter');
 
 const HtmlSplitter = polymerBuild.HtmlSplitter;
 const PolymerProject = polymerBuild.PolymerProject;
 const uglify = composer(uglifyes, console);
 
-const logging = require('plylog');
+// const logging = require('plylog');
```
Comment (sophieH29) [inline gulpfile.js]: maybe just remove this, if we don't use it?

File: gulpfile.js [hunk 2]
```diff
@@ -10,46 +12,51 @@ const mergeStream = require('merge-stream');
...
 // logging.setVerbose();
 
 const config = {
   polymerJsonPath: './polymer.json',
   build: {
-    rootDirectory: 'build'
+    rootDirectory: 'build',
```
Comment (sophieH29) [inline gulpfile.js]: eslint requires comma after last property in object? What a shit..

File: gulpfile.js (hunk 1 above)
Comment (ozasadnyy) [reply gulpfile.js]: we can use it if we want to get logs, so I would rather keep it for now

File: gulpfile.js (hunk 2 above)
Comment (ozasadnyy) [reply gulpfile.js]: Airbnb and Google styleguides promote this + here is a good article about benefits: https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8

**Pull request merged**